backend/FwLite/MiniLcm/Validators/JsonPatchValidator.cs

```diff
@@ -0,0 +1,22 @@
+﻿using FluentValidation;
+using SystemTextJsonPatch;
+
+namespace MiniLcm.Validators;
+
+internal static class JsonPatchValidator
+{
+    public static IRuleBuilderOptionsConditions<UpdateObjectInput<T>, JsonPatchDocument<T>> NoOperation<T>(
```

Comment:
Not a big deal, but I find the name a bit confusing.
Here's a potential non-commitable alternative(s). 😉 

```suggestion
    public static IRuleBuilderOptionsConditions<UpdateObjectInput<T>, JsonPatchDocument<T>> (Must|Does)NotTouch[Property]<T>(
```

Reply:
Yeah I wasn't sure what to do here. I think I'll do `DoesNotChange`